LSPS0/README.md

```diff
@@ -525,6 +573,91 @@ payload, indicating it supports LSPS1 and LSPS3 (in addition to LSPS0):
 }
 ```
 
+### LSPS Extension And Versioning
+
+Individual LSPS SHOULD NOT include their own explicit versioning
+scheme.
+
+An individual LSPS MAY be revised to a new version.
+Newer versions of an individual LSPS MUST NOT break compatibility with
+earlier versions.
+
+If there is a need for a change that breaks compatibility with earlier
+versions of an LSPS, then a new LSPS with a different LSPS number MUST
+be used.
+
+Newer versions of an LSPS MAY, if a new need arises:
+
+* Add new optional `params` fields for a client-callable API `method`.
+  * If the optional parameter is not specified, the `method` MUST
+    act the same as in previous versions before the parameter
+    existed.
+* Add new required or optional `result` fields for a client-callable
+  API `method`.
+  * Provided that the client is free to ignore the added fields and
+    that acceptance of a new `result` field is signalled by using a
+    new optional `params` field of some client-callable API `method`,
+    or by calling a new client-callable API `method`, or by some other
+    method, and that a lack of signal of acceptance of the new field
+    results in behaving the same as in older versions.
+    (**Rationale** Older clients WILL ignore unknown fields and thus
+    would not be aware of them)
+  * Allowed recursively for any field that itself contains a
+    dictionary (i.e. a `result` field may contain a dictionary, and a
+    new version of an LSPS may define a new field for that dictionary)
+    to any depth of dictionary fields, provided the previous provision
+    is followed.
+* Add new error `code`s for a client-callable API `method`.
+* Add completely new client-callable API `method`s.
+* Add completely new LSP-initiated notification `method`s, provided
+  they are enabled only by a new optional field in some
+  client-callable API `method` or by a completely new client-callable
+  API `method`.
+
+Newer version of an LSPS MUST NOT make changes beyond the above.
+
+Clients MUST:
+
+* Ignore unrecognized `result` fields from the result of a
+  client-callable API `method`.
+  * Recursively ignore unrecognized fields from any dictionaries
+    in a recognized `result` field, to any depth.
+* Treat as a generic error, and log as unusual, any unrecognized
+  error `code` from an `error` result for a client-callable
+  API `method`.
+* Ignore, but log as unusual, unrecognized LSP-initiated
+  notification `method`s.
```

Comment:
NIT

`MUST` is fierce wording here. Most users don't even realize their mobile app has logs. 
I wouldn't consider a client non-compliant because of a poor logging implementation.

Reply:
So still MUST ignore but remove the bit about logging?

Reply:
Put the logging bits as `SHOULD`.